building_energy_standards_data/applications/create_openstudio_standards_json.py

```diff
@@ -302,44 +303,59 @@ def create_openstudio_standards_space_data_json(
 
 
 def create_openstudio_standards_data_json_ashrae_90_1(
-    conn: sqlite3.Connection, version_90_1: str
+    conn: sqlite3.Connection,
+    version_90_1: str,
+    osstd_repository_path: str,
+    prm: bool = False,
```

Comment:
@leijerry888 - New PRM flag so the PRM can be generated as well. See example below.

```python
import sqlite3
conn = sqlite3.connect('openstudio_standards.db')
from building_energy_standards_data.applications.create_openstudio_standards_json import create_openstudio_standards_data_json_ashrae_90_1
for t in ["2019"]:
    create_openstudio_standards_data_json_ashrae_90_1(conn=conn, version_90_1=t, osstd_repository_path="../OSSTD-T1.1", prm=True)
conn.close()```